server/webhookNotifier.ts

```diff
@@ -35,6 +36,9 @@ const getFilteredContent = (content, address) => {
       labelerFn = CompoundEvents.Label;
     } else if (chainSupportedBy(content.chainEventType.chain, AaveEvents.Types.EventChains)) {
       labelerFn = AaveEvents.Label;
+    } else if (content.chainEvent.event_data.kind === 'approval' || content.chainEvent.event_data.kind === 'transfer') {
+      // TODO: improve this ERC20 detection
```

Comment:
We might just have to outline and make the hierarchy of inherited event kinds so that they can get if/then checked in appropriate order.